zer0-os/zOS
# feat: add zApp DAOs

### What does this do?

Adds zApp DAOs to the list of apps.

### Why are we making this change?

To get DAOs zApp in zOS for iteration.

### How do I test this?
- Click "DAOs" in the app selector.
- Check that the "DAOs" app renders.

### Key decisions and Risk Assessment:
  #### Things to consider:
  1. How will this affect security?
  1. How will this affect performance?
  1. Does this change any APIs?
